src/Storage/Azure.Storage.psd1

```diff
@@ -115,7 +115,9 @@ CmdletsToExport = 'Get-AzureStorageTable', 'New-AzureStorageTableSASToken',
                'Set-AzureStorageContainerStoredAccessPolicy', 
                'Start-AzureStorageBlobCopy', 
                'Start-AzureStorageBlobIncrementalCopy', 
```

Comment:
Please add tests for your new cmdlets.

Reply:
As the Storage Client library request can't be record in Unit test framework, generally we don't add unit test in PowerShell solution for data plan cmdelts. We will add functional test for the new cmdelts in storage team regression test. Let me know if you would like to review the CR for the our team regression test.

Reply:
I'd love to review the regression tests before this change goes into our repo.

Reply:
Hi Maddie,

Following is the CR link for the test cases. I have sent you another mail for the CR and attached the test log. You can see the powershell cmdelts tested and result validation of the test cases in the test log.

>codeflow://open/?server=https%3A%2F%2Fmsazure.visualstudio.com%2F&review=One.664474&host=vso&alert=true

Reply:
@maddieclayton 

I have added the Unit test for the service properties parse function as we discussed into the PR.
Please help to review.